web-client/src/ustc-ui/Tabs/Tabs.tsx

```diff
@@ -62,8 +55,9 @@ const renderTabFactory = ({
 
     return (
       <li {...tabProps}>
+        {' '}
         <button {...buttonProps} data-testid={child.props['data-testid']}>
-          <HeadingElement className="button-text">{title}</HeadingElement>{' '}
+          <span className="button-text">{title}</span> {icon}
```

Comment:
One side effect of this change: the slight difference in styles between the tabs on, e.g., the advanced search page vs. the message page will be removed. I discussed this and got approval from UX.